app/scripts/lib/config-loader.js

```diff
@@ -58,16 +56,16 @@ define([
         // did not receive the cookie.
       }
 
-      self._configPromise = self._xhr.getJSON('/config')
-        .fail(function (jqXHR) {
+      this._configPromise = this._xhr.getJSON('/config')
+        .fail((jqXHR) => {
           throw ConfigLoader.Errors.normalizeXHRError(jqXHR);
         });
 
-      return self._configPromise;
-    },
-  };
+      return this._configPromise;
+    }
+  }
 
-  var t = function (msg) {
+  let t = function (msg) {
```

Comment:
I kinda think all these random t's should just be put into its own module that we include where needed.